src/components/playlist/playlist.js

```diff
@@ -22,14 +22,17 @@ export const Audio = ({url$}) => url$.scan((last, src) => {
   if (canPlay({last, src})) return {src, type: 'PLAY'}
   return {src, type: 'PAUSE'}
 }, null)
-const view = ({playlistDOM$}) => {
-  return playlistDOM$.startWith(
-    <div>{P.PlaylistItem}{P.PlaylistItem}{P.PlaylistItem}</div>
-  ).map(view =>
-    <div className={css.playlist}>
-      {view}
-    </div>
-  )
+
+const view = ({playlistDOM$, seeking$}) => {
+  return playlistDOM$
+    .combineLatest(seeking$)
+    .startWith(
+      [<div>{P.PlaylistItem}{P.PlaylistItem}{P.PlaylistItem}</div>, false]
+    ).map(([view, seeking]) =>
+      <div className={css.playlist} style={{overflow: seeking ? 'hidden' : 'auto'}}>
```

Comment:
Create a new class — `disable-scroll` and simply add-remove it from the list.
http://www.quirksmode.org/dom/classchange.html